libs/partners/mistralai/langchain_mistralai/embeddings.py

```diff
@@ -84,13 +110,17 @@ def embed_documents(self, texts: List[str]) -> List[List[float]]:
             List of embeddings, one for each text.
         """
         try:
-            embeddings_batch_response = self.client.embeddings(
-                model=self.model,
-                input=texts,
+            batch_responses = (
+                self.client.embeddings(
```

Comment:
Do we have no mapping from source doc to embedding at this point if we're flat mapping the outputs? that seems undesirable. I'd probably average the embeddings or if we had been able to change the return type, return objects